judge/views/submission.py

```diff
@@ -159,7 +160,7 @@ class SubmissionsListBase(DiggPaginatorMixin, TitleMixin, ListView):
     context_object_name = 'submissions'
     first_page_href = None
 
-    def get_result_table(self):
+    def get_result_data(self):
         return get_result_table(self.get_queryset().order_by())
```

Comment:
`get_results_table`, I suppose if you're changing this better be thorough

Reply:
k